packages/icons/manifest.json

```diff
@@ -296,6 +296,11 @@
       "category": "Payment method",
       "size": "24"
     },
+    {
+      "name": "eftpos",
+      "category": "Payment method",
+      "size": "24"
+    },
```

Comment:
Please put the icon in alphabetical order, as specified in the [contribution guidelines](https://circuit.sumup.com/?path=/docs/contributing-contributing-icons--docs).